src/api/tests/test_llm.py

```
@@ -70,3 +70,52 @@ def fake_completion(*args, **kwargs):
     assert captured_kwargs["kwargs"]["temperature"] == 0.7
     assert captured_kwargs["kwargs"]["stream"] is True
     assert span.generation_args["name"] == "chat-test"
+
+
+def test_invoke_llm_ark_endpoint_uses_model_name_for_max_tokens(monkeypatch, app):
+    captured_kwargs = {}
+    captured_token_models = []
+
+    def fake_completion(*args, **kwargs):
```

Ruff (0.14.10)

Unused function argument: `args`

(ARG001)